src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -578,69 +578,36 @@
-            --git-access-token 0000000000000000000000000000000000000000 --platform linux/arm/v7
-  - name: Create a Linux task from a public GitHub repository which builds the hello-world image with a git commit trigger. Note that this task does not use Source Registry (MyRegistry), so we can explicitly set Auth mode as None for it.
+            -c https://msazure.visualstudio.com/DefaultCollection/Project/_git/Repo#Branch:Folder \\
+            -f Dockerfile --git-access-token <Personal Access Token> --platform linux/arm/v7
+  - name: Create a Linux task from a public GitHub repository which builds the hello-world image with only a git pull request trigger. Note that this task does not use Source Registry (MyRegistry), so we can explicitly set Auth mode as None for it.
     text: |
-        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry \\
-            --auth-mode None -c https://github.com/Azure-Samples/acr-build-helloworld-node.git -f Dockerfile \\
-            --git-access-token 0000000000000000000000000000000000000000
-  - name: Create a Linux task from a public GitHub repository which builds the hello-world image with a git pull request trigger. Note that this task does not use Source Registry (MyRegistry), so we can explicitly set Auth mode as None for it.
-    text: |
-        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry \\
-            --auth-mode None -c https://github.com/Azure-Samples/acr-build-helloworld-node.git -f Dockerfile \\
-            --pull-request-trigger-enabled True --git-access-token 0000000000000000000000000000000000000000
+        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry  -f Dockerfile \\
+            --auth-mode None -c https://github.com/Azure-Samples/acr-build-helloworld-node.git \\
+            --commit-trigger-enabled false --git-access-token 00000000000000000000000000000000000000
   - name: Create a Windows task from a public GitHub repository which builds the Azure Container Builder image on Amd64 architecture without triggers.
     text: |
         az acr task create -t acb:{{.Run.ID}} -n acb-win -r MyRegistry \\
             -c https://github.com/Azure/acr-builder.git -f Windows.Dockerfile \\
-            --commit-trigger-enabled false --platform Windows/amd64
-  - name: Create a Linux multi-step task from a public GitHub repository with a git commit trigger and with the system-assigned managed identity
-    text: |
-        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry \\
-            -c https://github.com/Azure-Samples/acr-tasks.git#:multipleRegistries -f testtask.yaml \\
-            --assign-identity
-  - name: Create a Linux multi-step task from a public GitHub repository with a git commit trigger and with a user-assigned managed identity
+            --commit-trigger-enabled false --pull-request-trigger-enabled false \\
```

Comment:
@rosanch ditto here on removing the pr trigger. Also let's include disabling the base image update trigger here as well as we specify "no triggers".